lib/generators/spree/uqpay/install/templates/check_payment_status_worker.rb

```diff
@@ -0,0 +1,25 @@
+class CheckPaymentStatusWorker
+  include Sidekiq::Worker
+  include UqpayCommon
+
+  def perform(*args)
+    orders = Spree::Order.where("payment_state = :payment_state and updated_at <= :updated_at", {payment_state: "balance_due", updated_at: 1.minutes.ago})
```

Comment:
It safer to query with `payment source date` instead of `updated_at`, because updated_at will change if admin change something on the order from admin

Reply:
Also the time should be `100.munutes.ago`